shared/src/commonMain/kotlin/api/person/absence/requestHours/HourRequestType.kt

```diff
@@ -0,0 +1,7 @@
+package api.person.absence.requestHours
+
+enum class HourRequestType(order: Int) {
+    Nothing(0),
```

Comment:
with caps i meant NOTHING (thats the standard, but this is fine too)

Reply:
want me to change it?

Reply:
you didnt reply so i did it anyway :sunglasses: